pkg/lib/clusterconfig/clusterconfig.go

```diff
@@ -74,10 +80,31 @@ var Validation = &cr.StructValidation{
 				Validator: validateInstanceType,
 			},
 		},
+		{
+			StructField:         "InstanceCPU",
```

Comment:
nit (maybe not a big deal, let's discuss): with this, I believe the user could add `instance_cpu` to their cluster config without creating a validation error (although it would be ignored).